examples/grid2grid/main.py

```diff
@@ -0,0 +1,161 @@
+from evasdk import Eva
+from evaUtilities import *

```

Comment:
I think you only need to import one or two functions here

Reply:
only importing EvaGrids now